### PR TITLE
Feature/update OCI docker documentation

### PR DESCRIPTION
**Pull request by daniel-warren**
Update the OCI tutorial with new steps following the change to the docker images.

New PR where GCP changes are not accidentally included.

### CODE REVIEW

File: _includes/cloud-docker/oraclecloud.md [hunk 1]
```diff
@@ -58,13 +115,8 @@ sudo service docker start
...
 Now launch the docker image.
 
-If you are using the trial:
-```
-sudo docker run -p 80:80 --env TOKEN=<TRIAL TOKEN> idrsolutions/{{ page.product | downcase }}
-```
-If you are using the full version:
 ```
-sudo docker run -p 80:80 --env LICENSE_USERNAME=<YOUR USERNAME> --env LICENSE_PASSWORD=<YOUR PASSWORD> --env PRODUCT=<YOUR CHOSEN PRODUCT> idrsolutions/{{ page.product | downcase }}
+sudo docker run -p 80:80 --mount "source=/home/opc/{{ page.product | downcase }}_microservice.war,target=/usr/local/tomcat/webapps/ROOT.war,type=bind" idrsolutions/{{ page.product | downcase }}
```
Comment (Amy1612) [inline _includes/cloud-docker/oraclecloud.md:119]: Just checking here, not too sure but does it have to be tomcat in the target path or can it be like jetty, glassfish, etc. If so would it be worth noting this to the customer, like mount to the web app you set up.

Comment (daniel-warren) [reply _includes/cloud-docker/oraclecloud.md:119]: A good question! As this guide is specifically for using our provided docker images ([JPedal ](https://github.com/idrsolutions/jpedal-docker/blob/master/Dockerfile)/ [BuildVu](https://github.com/idrsolutions/buildvu-docker)) we can expect a set environment. If you check the Dockerfile in either project you'll see that the image is based on a Tomcat docker image, as such we only expect tomcat for the target path (at present the other app servers are not installed in the docker image).

Comment (Amy1612) [reply _includes/cloud-docker/oraclecloud.md:119]: ahh okay, thank you for explaining!

File: _includes/cloud-docker/oraclecloud.md [hunk 2]
```diff
@@ -2,43 +2,100 @@
...
+<p>&nbsp;</p>
+
+**For customers:**
+1. Visit the [customer download page][customer-download] for {{ page.product }}.
+
+2. Click on 'Download Latest Customer WAR'.
+
+3. Enter your username and password.
+
+4. Click submit.
```
Comment (leonatherton) [inline _includes/cloud-docker/oraclecloud.md:23]:
```suggestion
3. Submit your username and password.
```

File: _includes/cloud-docker/oraclecloud.md [hunk 3]
```diff
@@ -80,38 +132,25 @@ Re-connect to your instance and add your certificate, key, and ca_bundle. They s
...
 
-### Verification [Required]
-These are used to verify your {{ page.product }} license type, either a token or username and password is required
-* TOKEN: used to verify your trial token.
-* LICENSE_USERNAME: used to verify your {{ page.product }} license.  
-* LICENSE_PASSWORD: used to verify your {{ page.product }} password.
-
-### Access
-These are used to set a username and password for your users to login with
+These are used to set a username and password for your users to login with:
```
Comment (leonatherton) [inline _includes/cloud-docker/oraclecloud.md:149]:
```suggestion
These are used to set a username and password to access the microservice:
```

**Review by leonatherton — CHANGES_REQUESTED** [review]

**Comment by daniel-warren** [comment]
@leonatherton - Suggested changes have been committed. Thanks for spotting.

**Review by leonatherton — APPROVED** [review]

**Review by Amy1612 — APPROVED** [review]

**Review by KieranFrance — APPROVED** [review]
Approved, as this is the last review please feel free to merge PR with develop branch

**Pull request merged**